src/Components/GameSession.js

```diff
@@ -72,10 +73,12 @@ export default function GameSession() {
     } else {
       importDefaultWords(randInts);
     }
+  }, [gameboardId, gamesStarted]);
 
+  useEffect(()=> {
     // daubers center tile
     dauberTile(12);
-  }, [gameboardId, gamesStarted]);
```

Comment:
Did you mean to remove the dependency array here? Without it, this useEffect hook will run on every render which shouldn't be necessary.

Reply:
Yes. `gameboardId` and `gameStarted` are no longer dependants to that `useEffect` so I removed them. Since refactoring, `dauberTile` method short-circuits if the time with matching ID is already true, so I figured it was an inexpensive call to ensure the FREE space gets daubered during each gameboard launch. Adding those dependencies back causes a linter warning. Suggestions?